test/e2e/test_job_submissions.py

```diff
@@ -735,10 +735,14 @@ def test_worker_reports_canceled_sync_input_actions_as_canceled(
         # Create the input files to make sync inputs take a relatively long time
         files_path: str = os.path.join(tmp_path, "files")
         os.mkdir(files_path)
-        for i in range(2000):
+        for i in range(6000):
             file_name: str = os.path.join(files_path, f"input_file_{i+1}.txt")
             with open(file_name, "w+") as input_file:
-                input_file.write(f"{i}")
+                if i % 1000 == 0:
```

Comment:
https://unix.stackexchange.com/questions/33629/how-can-i-populate-a-file-with-random-data

This seems to be an easier solution?

Reply:
We don't need random data, we need a lot of small files (to ensure the syncInputJobAttachments takes a long time) and a few large files (to ensure the syncInputJobAttachments does not fail because of the perceived low transfer rail)

Reply:
We also cannot assume that the tests are running on a machine with `/dev/random` and `dd` available. If the host running the tests is a Windows machine without WSL, it will not have those.